db/migrate/20250626105725_add_library_types_for_npg_pipeline.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
```

Comment:
I would have used RecordLoader for this, by adding a file in config/default_records/library_types.